### DISCUSSION TITLE
Reset admin passwd

### DISCUSSION THREAD

**Discussion by dasc19** [photoprism/photoprism#3578] [Q&A]
Hi there

I missed up my bitwarden server and I loose my admin passwd .I have a docker compose installation. Is there any way to reset  the admin passwd?

Thanks in advance for any support 

Rgds

**Comment by graciousgrey** [comment]
Hi :)

You can use the change password command for this: https://docs.photoprism.app/getting-started/docker-compose/#command-line-interface

**Comment by dasc19** [comment]
Hi There

Thanks a  lot 

It's done
